### PR TITLE
2539 template download

### PR DESCRIPTION
**Pull request by sanjuroj**

### CODE REVIEW

File: indicators/views/bulk_indicator_import_views.py [hunk 1]
```diff
@@ -29,15 +29,18 @@
...
+    {'name': 'No.', 'required': False, 'field_name': 'number'},
     {'name': 'Indicator', 'required': True, 'field_name': 'name'},
     {'name': 'Sector', 'required': False, 'field_name': 'sector',
      'validation': VALIDATION_KEY_SECTOR},
     {'name': 'Source', 'required': False, 'field_name': 'source'},
     {'name': 'Definition', 'required': False, 'field_name': 'definition'},
     {'name': 'Rationale or justification for indicator', 'required': False, 'field_name': 'justification'},
     {'name': 'Unit of measure', 'required': True, 'field_name': 'unit_of_measure'},
-    {'name': 'Number (#) or percentage (%)', 'required': True, 'field_name': 'unit_of_measure_type',
+    # Note:  this lone string is being translated here because it's not the
```
Comment (andrethomas6) [inline indicators/views/bulk_indicator_import_views.py:40]: Did this note get cut off?

File: indicators/views/bulk_indicator_import_views.py [hunk 2]
```diff
@@ -250,7 +257,16 @@ def get(self, request, *args, **kwargs):
 
             # Add notes to header row cells
             if header['field_name'] != 'comments':
-                help_text = Indicator._meta.get_field(header['field_name']).help_text
+                if header['field_name'] == 'number' and not program.auto_number_indicators:
+                    # Translators: This is help text for a form field that gets filled in automatically
+                    help_text = gettext('This number is automatically generated through the results framework')
```
Comment (andrethomas6) [inline indicators/views/bulk_indicator_import_views.py:262]: Just missing a period at the end.

**Review by andrethomas6 — APPROVED** [review]
Looks good, just a couple of little comments.

File: indicators/views/bulk_indicator_import_views.py (hunk 1 above)
Comment (sanjuroj) [reply indicators/views/bulk_indicator_import_views.py]: heh, forgot to finish typing it

File: indicators/views/bulk_indicator_import_views.py (hunk 2 above)
Comment (sanjuroj) [reply indicators/views/bulk_indicator_import_views.py]: Thx!

**Pull request merged**